src/content/1.2/code/scala/snippet01.scala

```diff
@@ -0,0 +1 @@
+val x: BigInt
```

Comment:
Just curious why `BigInt` here and in other places?

Reply:
I suppose it was to convey the fact we're dealing with an infinite (or, very large) number. The text below explains:

> `Integer` in Haskell is an infnite set, and it can be used to do arbitrary precision
arithmetic. Tere is also a fnite-set `Int` that corresponds to machine
type, just like the C++ `int`.